pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -321,7 +321,7 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 	container.Resources = corev1.ResourceRequirements{
 		Requests: corev1.ResourceList{
 			corev1.ResourceCPU:    resource.MustParse("10m"),
-			corev1.ResourceMemory: resource.MustParse("325Mi"),
+			corev1.ResourceMemory: resource.MustParse("357Mi"),
```

Comment:
What do you think about adding a comment here with the info where this number comes from? E.g. smth like this:
```suggestion
			// 325Mi - base memory request
			// +32Mi - to account for the buffer used to verify containerdisk checksums
			corev1.ResourceMemory: resource.MustParse("357Mi"),
```
With that, if someone looks at the code later, it will not be needed to search through the history in git.

Reply:
sounds fair, it costs nothing to add it :)

Reply:
@vasiliy-ul lgtm got removed as I added this, thanks for the review!

Reply:
Interesting, we can be doing more than one verification or not? I think this is a small peak in memory and we don't necessarily need to account for it in requests.

Reply:
How is this one at a time?

Reply:
@xpivarc Sorry you're right, it's at most "virt-handler-threads" (10) verifications in parallel, I'm not sure what we should account for, maybe just adding 1 buffer it's an ok assumption? The verification is so quick that I'd be surprised if we get oom-killed for this. Ideas?

Reply:
Good point about parallel verification :+1:  As I checked now, we do not set memory limits for virt-handler. So if the host has enough memory, all should be good. If not, then there is a chance that it will be OOM killed, but I think it is hard to find a proper value as it depends on many factors. The memory request AFAIK affects mostly the scheduling of a pod, i.e. it defines the 'minimal' amount of memory required. Perhaps accounting for just one buffer would be fine as a starting point.